usages.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+const (
+	litmusBanner = `
```

Comment:
Proper fancy, I like!!

Reply:
ANSI Codes for the win, it displays nicely in terminal output, but only when you call `-h` flag, don't want to poop over everything each time you use it

Reply:
Totally agree, it's quite ... exciting :P  I like that it's only displayed in the help text.  It'll probably be the first thing the user sees and then never sees again.  Which is perfect.